src/tirith/core/core.py

```diff
@@ -56,30 +56,43 @@ def generate_evaluator_result(evaluator_obj, input_data, provider_module):
     evaluation_results = []
     has_evaluation_passed = True
 
-    for evaluator_input in evaluator_inputs:
-        if isinstance(evaluator_input["value"], ProviderError) and evaluator_input.get("err", None):
-            severity_value = evaluator_input["value"].severity_value
-            err_result = dict(message=evaluator_input["err"])
-
-            if severity_value > evaluator_error_tolerance:
-                err_result.update(dict(passed=False))
+    # If there are no evaluator inputs, it means the provider didn't find any resources
+    # In this case, the evaluation should fail
+    if not evaluator_inputs:
+        has_evaluation_passed = False
+        evaluation_results = [{"passed": False, "message": "Could not find input value"}]
```

Comment:
Could you add this in here:
```
    result["result"] = evaluation_results
    result["passed"] = has_evaluation_passed
    return result
```

and then remove the `else` below and deindent, just for readability

Reply:
Will followup later on this